OnBoarding/Configuration.php

```diff
@@ -177,7 +177,7 @@ public function getConfiguration(Router $router)
                         'savepoint',
                     ],
                     'page' => $contextLink->getAdminLink('AdminPayment'),
-                    'selector' => '.modules_list_container_tab:first tr:first-child .text-muted',
+                    'selector' => '.modules_list_container_tab:first tr:first-child .text-muted, .card:eq(0) .text-muted:eq(0)',
```

Comment:
is the former selector still useful then?

Reply:
Yes, the existing selector is kept for older versions of PrestaShop 1.7. By doing this, we don't have to maintain one more version of the same module.

Reply:
alright, smart!